src/23680.c

```diff
@@ -168,13 +169,13 @@ s32 func_800490B4(EnemyTerritoryThing* territory, Enemy* enemy, f32 radius, f32
         }
     }
 
-    if (territory->unk_00 == 0) {
-        phi_v0 = 0;
+    if (territory->skipPlayerDetectChance == 0) {
+        skipCheckForPlayer = FALSE;
     } else {
-        phi_v0 = rand_int(territory->unk_00 + 1);
+        skipCheckForPlayer = rand_int(territory->skipPlayerDetectChance + 1);
     }
 
-    if (phi_v0 == 0) {
+    if (!skipCheckForPlayer) {
```

Comment:
I'd rather keep `skipCheckForPlayer == 0` and `0` because of the `rand_int` usage; it's not really just a bool. You also use `0` in `func_800493EC` instead of `FALSE`